cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -30,4 +34,32 @@ public boolean install(boolean silent) {
     return super.install(silent);
   }
 
+  @Override
+  public void installPlugin(PluginDescriptor plugin) {
+
+    Path mavenPlugin = this.context.getSoftwarePath().resolve(this.tool).resolve("lib/ext/" + plugin.getName() + ".jar");
+    System.out.println(getPluginUrl(plugin));
+    this.context.getFileAccess().download(getPluginUrl(plugin), mavenPlugin);
+    if (Files.exists(mavenPlugin)) {
+      this.context.success("Successfully added {} to {}", plugin.getName(), mavenPlugin.toString());
+    } else {
+      this.context.warning("Plugin {} has wrong properties\n" //
+          + "Please check the plugin properties file in {}", mavenPlugin.getFileName(), mavenPlugin.toAbsolutePath());
+    }
+  }
+
+  private String getPluginUrl(PluginDescriptor plugin) {
+
+    if (plugin.getUrl() != null) {
+      return plugin.getUrl();
+    } else {
+      return "https://repo1.maven.org/maven2/fr/jcgay/maven/" + plugin.getName() + "/" + getPluginVersion(plugin.getVersion()) + "/" + plugin.getName()
+          + "-" + getPluginVersion(plugin.getVersion()) + ".jar";
```

Comment:
this is also incorrect as you are hardcoding a `groupId` (`fr.jcgay.maven`) here that will be different for each plugin.
As stated above the most simple way to get this done is to just make the URL mandatory for the moment and kick out the version and this code.
Then we can later add a PR to add this feature enhancement properly.